web/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "alman-akka",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "12.1.0",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
+  },
+  "devDependencies": {
+    "@types/node": "17.0.21",
+    "@types/react": "17.0.39",
+    "eslint": "8.10.0",
+    "eslint-config-next": "12.1.0",
+    "eslint-config-prettier": "^8.5.0",
```

Comment:
Some of these package versions are "hard-coded" and some use caret (^). Don't know if it has effect on anything e.g. when using dependabot 🤔

Reply:
Dependabot should know how to upgrade these. ✅